xs2a-adapter-aspsp-registry/src/main/java/de/adorsys/xs2a/adapter/registry/LuceneAspspRepository.java

```diff
@@ -42,15 +45,61 @@ public void save(Aspsp aspsp) {
         }
     }
 
+    @Override
+    public Aspsp create(Aspsp aspsp) {
+        Optional<Aspsp> optional = findById(aspsp.getId());
```

Comment:
Calling `findById` with a null will result in NPE

Reply:
thanks, fixed